### PR TITLE
CB-8976: Removing the auto-version for non-Crosswalk applications

### PR DESCRIPTION
**Pull request by infil00p**
Discussed this on the mailing list, going to try to not just toss this over the fence.

### CODE REVIEW

File: bin/templates/project/build.gradle [hunk 1]
```diff
@@ -86,6 +86,7 @@ ext {
 }
 
 // PLUGIN GRADLE EXTENSIONS START
+apply from: "cordova-plugin-crosswalk-webview/app-xwalk.gradle"
```
Comment (riknoll) [inline bin/templates/project/build.gradle]: Is this import intentional?

Comment (infil00p) [inline bin/templates/project/build.gradle]: Nope, I'll remove it and redo the pull request.

Comment (riknoll) [inline bin/templates/project/build.gradle]: Alright, I think that lines 249 and 250 might have been added in as well

**Comment by nikhilkh** [comment]
LGTM.

**Comment by csantanapr** [comment]
LGTM

File: bin/templates/project/build.gradle [hunk 2]
```diff
@@ -197,7 +197,12 @@ android {
                 }
             }
         }
-    } else if (!cdvVersionCode) {
+    }
+    /*
+
+    ELSE NOTHING! DON'T MESS WITH THE VERSION CODE IF YOU DON'T HAVE TO!
```
Comment (colonelchlorine) [inline bin/templates/project/build.gradle]: Best piece of code ever written. Why is Cordova doing this in the first place?

Comment (nikhilkh) [inline bin/templates/project/build.gradle]: If you're building multiple apks for different architectures (for Crosswalk) - there is a need to generate unique version codes.

**Pull request closed without merging**